File: backend/src/main/java/br/com/ifsp/classify/dtos/create/ClassSessionCreateDTO.java
package br.com.ifsp.classify.dtos.create;

import java.time.LocalDateTime;

public record ClassSessionCreateDTO(
        String subjectTeacherUuid,
        String classroomUuid,
        LocalDateTime startTime,
        LocalDateTime endTime,
        ReportCreateDTO report,
        String classUuid,
        String studentUuid
) {}

File: frontend/src/shared/dtos/class-session/ClassSessionCreateDTO.ts
import type { ReportCreateDTO } from "../report/ReportCreateDTO";

export interface ClassSessionCreateDTO {
    subjectTeacherUuid: string;
    classroomUuid: string;
    startTime: string;
    endTime: string;
    report: ReportCreateDTO;
    classUuid: string;
    studentUuid: string;
}
